.hound.yml

```diff
@@ -0,0 +1,2 @@
+swiftlint:
+  config_file: .swiftlint.yml
```

Comment:
`.swiftlint.yml` seems to be detected automatically...

Reply:
Looks like HoundCI uses their default config if it is not specified explicitly. So 👍 on this.